src-opam/distro.ml

```diff
@@ -542,6 +545,7 @@ let distro_status (d : t) : status =
         `Deprecated
     | `Alpine `V3_14 -> `Active `Tier2
     | `Alpine `V3_15 -> `Active `Tier1
+    | `Alpine `V3_16 -> `Active `Tier1
```

Comment:
Note sure about this. I read the documentation on tiers but I still don't know if that's the correct tier to set for the alpine versions.

Reply:
3.15 should probably be downgraded to Tier2. Otherwise this would create all the variant images for both 3.15 and 3.16